x-pack/plugins/index_lifecycle_management/public/sections/policy_table/components/policy_table/policy_table.js

```diff
@@ -314,7 +314,6 @@ export class PolicyTableUi extends Component {
     const button = (
       <EuiButtonEmpty
         data-test-subj="policyActionsContextMenuButton"
-        aria-label="Policy options"
```

Comment:
@bmcconaghy Do you think we still need this? I figure the text in the button itself is explanatory ("Actions"). Otherwise, I'd suggest we change this to be "Policy actions".

```jsx
        aria-label={intl.formatMessage({
          id: 'xpack.indexLifecycleMgmt.policyTable.actionsButtonAriaLabel',
          defaultMessage: 'Policy actions',
        })}
```